src/runtime/propensity-test.js

```diff
@@ -126,24 +128,28 @@ describes.realWin('Propensity', {}, env => {
     }).to.throw('publisher not whitelisted');
   });
 
-  it('should send event params to server', () => {
+  it('should send events to event manager', () => {
     let eventSent = null;
-    let paramsSent = null;
-    const params = /** @type {JsonObject} */ ({'source': 'user-action'});
-    const eventParams = JSON.stringify(params);
-    const propensityEvent = {
-      name: PropensityApi.Event.IMPRESSION_OFFERS,
+    const params = /** @type {JsonObject} */ ({
+      'source': 'user-action',
+      'is_active': false,
```

Comment:
lets remove is_active and the value for 'source' to email.

Reply:
ok